components/common/src/command/package/install.rs

```diff
@@ -229,6 +260,7 @@ impl<'a> InstallTask<'a> {
     ) -> Result<Self> {
         Ok(InstallTask {
             depot_client: Client::new(url, product, version, Some(fs_root_path))?,
+            channel: channel,
```

Comment:
Can this (and the lines around it) make use of [field init shorthand](https://doc.rust-lang.org/book/second-edition/ch05-01-defining-structs.html#using-the-field-init-shorthand-when-variables-and-fields-have-the-same-name)?